packages/pebble-shared/icons.json

```diff
@@ -1,4 +1,12 @@
 [
+  "Edit note",
+  "Hidden",
+  "Lock",
+  "Report",
+  "Schedule mail",
+  "System",
+  "Unlock",
+  "actions",
```

Comment:
icon names should be all lowercase and hyphen separated
example: "Edit note" should be "edit-note"

Reply:
done